src/users/views.py

```diff
@@ -80,18 +80,18 @@ class CreateUser(generics.CreateAPIView):
     
     @transaction.atomic
     def post(self, request):
-        if User.objects.filter(email=request.data['email']).exists():
+        if User.objects.filter(email=request.data['email'].lower()).exists():
```

Comment:
use this: 
https://stackoverflow.com/questions/11743207/django-model-case-insensitive-query-filtering

Reply:
also, i'd like to see a test case to ensure this behavior exists after people edit code